NAV-154-task-list-page

Manoj-nathwani opened the pull request:
https://fjelltopp.atlassian.net/browse/NAV-154

![image](https://user-images.githubusercontent.com/2634482/143265168-1c12f2b9-3d31-4ab4-8a9e-2f09d800bf57.png)

jonathansberry reviewed (changes requested):
I think we need a button "Refresh Task List" or something similar?

What happens for long task titles? Can we (a) make the task list area wider, and (b) test the display of multi line task titles (i.e. Does the text indentation work correctly around the check box).

Can we be consistent with the what's next page in the checkboxes we are using (especially regarding empty, filled in and checked variants of the icon).

We need a little "More tasks may be added" message.  I've been thinking we should do this on a per milestone basis and on a overall basis. This requires us adding in a "fullyResolved" param for each milestone and for the overall task list. I'll speak to Tomek about this, but you should already have the "overall" fullyResolved param in the stub Tomek has created you?

Manoj-nathwani commented:
@jonathansberry all your above comments have now been addressed ✅ 

Added Refresh button  & made tasks wider

![image](https://user-images.githubusercontent.com/2634482/143452129-48e03c56-df72-4386-80ae-4cce9985c08b.png)


Checkbox and text now wraps nicely 

![image](https://user-images.githubusercontent.com/2634482/143452067-3c9e86c3-0db5-4b7d-93aa-7f9d35c3e549.png)


Milestones listview and tasks list pages now have consistent icons, margins, etc
The same react component is being reused for both pages

![image](https://user-images.githubusercontent.com/2634482/143450666-852206d3-0733-4f3c-971b-e0c872fe6693.png)

Added `More tasks may be added` message if `fullyResolved:false`

![image](https://user-images.githubusercontent.com/2634482/143451820-c5f11d84-75e4-4ff5-b21e-2c57d2348fc1.png)
